torch/csrc/distributed/rpc/ProcessGroupAgent.cpp

```diff
@@ -73,14 +73,22 @@ ProcessGroupAgent::ProcessGroupAgent(
       "Resolved worker rank ", workerRankIter -> second,
       " does not match ProcessGroup rank ", pg_->getRank());
 
-  names_.resize(nameMap_.size());
-  for (auto& entry : nameMap_) {
-    names_[entry.second] = entry.first;
-  }
   sendThread_ = std::thread(&ProcessGroupAgent::sendLoop, this);
   listenerThread_ = std::thread(&ProcessGroupAgent::listenLoop, this);
 }
 
+worker_id_t ProcessGroupAgent::getId() {
```

Comment:
Can we call this method getWorkerId() too? just does not take argument.

Reply:
Oh. Checked Pieter's comment. Seems he has a different naming convention.

Reply:
> Will this bring challenge to pybind ?

I should be able to do `py::overload_cast` to merge these two functions into one.